terraform/static/assets/cui-2.0.0/js/functionLibrary.js

```diff
@@ -137,3 +137,42 @@ function loadInputLimit(limitArr) {
   }
   // console.log("Loaded saved inputs from local storage.");
 }
+
+let previousScrollTop = 0;
+function autoScroll(input) {
+  const frame = document
+    .querySelector("iframe")
+    .contentWindow.document.querySelector("html");
+  const bottom = frame.scrollHeight;
+  if (frame.scrollTop == previousScrollTop || input !== null) {
+    frame.scrollTo(0, bottom);
+    previousScrollTop = frame.scrollTop;
+  }
+}
+
+let autoScrollTimer = null;
+function startAutoScroll() {
+  autoScrollTimer = setInterval(() => {
+    autoScroll();
+    console.log("autoScrolled")
+  }, 200)
+}
+
+function endAutoScroll() {
+  if (autoScrollTimer !== null)
+    clearInterval(autoScrollTimer);
+}
+
+let autoScrollBool = true;
+function toggleAutoScroll() {
+  autoScrollBool = !autoScrollBool;
+  autoScrollButton = document.getElementById("autoScrollToggle");
```

Comment:
`const autoScrollButton `
Best practice is to always declare variables with var, let or const.
You may have just missed this one.  😸

Reply:
Nice catch! Will fix that

Reply:
I added const there, is fixed now in latest commit